jason/tests/peer/mod.rs

```diff
@@ -2,15 +2,24 @@
 
 mod media;
 
-use std::rc::Rc;
+use std::{cell::RefCell, rc::Rc};
+
+use futures::{
+    future::{self, IntoFuture},
+    sync::mpsc::{unbounded, UnboundedReceiver},
```

Comment:
Do not hesitate to use `mpsc::unbounded` instead of just `unbounded`. It's still quite ergonomic while much more informative.